### PR TITLE
Test

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
Follows https://github.com/ja-che/hidimstat/pull/25

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
Once again, only the MEG example and Circle is green:

https://36-259908992-gh.circle-artifacts.com/0/dev/index.html

The example:

https://36-259908992-gh.circle-artifacts.com/0/dev/auto_examples/plot_meg_data_example.html#sphx-glr-auto-examples-plot-meg-data-example-py

**Comment by GuillaumeFavelier** [comment]
```
❯ git grep Parallel
examples/plot_fig_1_nguyen_et_al.py:from joblib import Parallel, delayed
examples/plot_fig_1_nguyen_et_al.py:    results = Parallel(n_jobs=1)(
hidimstat/desparsified_lasso.py:from joblib import Parallel, delayed
hidimstat/desparsified_lasso.py:        Parallel(n_jobs=n_jobs, verbose=verbose)(
hidimstat/knockoffs/knockoff_aggregation.py:from joblib import Parallel, delayed
hidimstat/knockoffs/knockoff_aggregation.py:    parallel = Parallel(n_jobs, verbose=joblib_verbose)
hidimstat/parallel.py:"""Parallel util function."""
hidimstat/parallel.py:        To pass to joblib.Parallel instance.
hidimstat/parallel.py:    parallel: instance of joblib.Parallel or list
hidimstat/parallel.py:            from joblib import Parallel, delayed
hidimstat/parallel.py:        parallel = Parallel(n_jobs, **kwargs)
hidimstat/permutation_test.py:from joblib import Parallel, delayed
hidimstat/permutation_test.py:        Parallel(n_jobs=n_jobs, verbose=verbose)(
```

**Comment by GuillaumeFavelier** [comment]
I migrared some, just `knockoffs` and `permutation_test` remaining I think

File: hidimstat/parallel.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+if 'MNE_FORCE_SERIAL' in os.environ:
+    _force_serial = True
+else:
+    _force_serial = None
+
+
+def parallel_func(func, n_jobs, **kwargs):
+    """Return parallel instance with delayed function.
+
+    Util function to use joblib only if available
```
[the remark this answers is not in the document]
Comment (agramfort) [reply hidimstat/parallel.py:20]: Joblib fails on CircleCI containers as we have seen in the past with mne. And 3D plots don’t work on GitHub actions. We have to choose between an up to date doc with good figures with this wrapper or no brain plots in the doc.

**Comment by GuillaumeFavelier** [comment]
It seems CircleCI continues to timeout even after `parallel_func()` and `MNE_FORCE_SERIAL`

https://app.circleci.com/pipelines/github/ja-che/hidimstat/45/workflows/6eaebd06-8b12-4e58-844b-47c1b75f8b0c/jobs/47

**Comment by agramfort** [comment]
closing in favor of https://github.com/ja-che/hidimstat/pull/30

**Pull request closed without merging**